docs/network_scenarios.md

```diff
@@ -0,0 +1,25 @@
+### Network scenario
+Scenario to introduce network latency, packet loss, bandwidth restriction in the Node's hostnework interface. The purpose of this scenario is to observe faults caused by random variations in the network.
+
+##### Sample scenario config
+```
+network_scenarios:                                  # Scenario to create an outage by simulating random variations in the network.
+  duration: 300                                     # seconds
+  node_name:                                        # node on which scenario has to be injected;
+  label_selector: node-role.kubernetes.io/master    # when node_name is not specified, a node with matching label_selector is selected for running the scenario.
```

Comment:
Maybe allow user to choose the node count to be able to inject the chaos on multiple nodes matching the label selector? Thoughts?

Reply:
will resubmit code with support to using more then one node for network chaos injection.